### ISSUE TITLE
add units in headers of our app default

### ISSUE THREAD

**Issue by gabrielareto** [Alliance-for-Tropical-Forest-Science/DataHarmonization#39]
I think it is better to add explicitly the units in the headers of our default format. It seems the default format will be used often. People forget about the possibility that they original units have changed. They keep the inertia of the original units. Everything is in metadata, and the metadata table is clearly a must. But most of our users infer variable content by looking at their data table in Excel -- consistent use of metadata tables or other forms of documentation are not as common as one would think.

long story short, having the units incorporated into the variable names can help.

**Comment by ValentineHerr** [comment]
Harder than expected. This is causing a lot of issues and it would take a while to implement.
How important is this? (note that the metadata has the units).

**Comment by ValentineHerr** [comment]
ok great, closing this for now then
